docs/source/package_reference/cli.md

```diff
@@ -228,6 +228,30 @@ The following arguments are only useful when training in SageMaker
 * `--aws_access_key_id AWS_ACCESS_KEY_ID` (`str`) -- The AWS_ACCESS_KEY_ID used to launch the Amazon SageMaker training job
 * `--aws_secret_access_key AWS_SECRET_ACCESS_KEY` (`str`) -- The AWS_SECRET_ACCESS_KEY used to launch the Amazon SageMaker training job
 
+## accelerate estimate-memory
+
+**Command**:
+
+`accelerate estimate-memory` or `accelerate-estimate-memory` or `python -m accelerate.commands.estimate`
+
+Estimates the total vRAM a particular model hosted on the Hub needs to perform Big Model Inference and training. Requires that `huggingface_hub` be installed.
```

Comment:
```suggestion
Estimates the total vRAM a particular model hosted on the Hub needs to perform inference and training. Requires that `huggingface_hub` be installed.
```

Reply:
Also, maybe we should add `huggingface_hub` as a core dep of Accelerate? It's meant to be the place of low-level utils shared across the ecosystem.

Reply:
In order not to mess with too much in a single PR, I'll add that in as a follow up